### ISSUE TITLE
2.14.1: Tests fail: REQUIRE( coordinates(1) == Approx(0.0f).margin(0.01) )

### ISSUE THREAD

**Issue by yurivict** [mlpack/ensmallen#225]
```
]$ ./work/.build/ensmallen_tests
ensmallen version: 2.14.0 (No Direction Home)
armadillo version: 9.900.2 (Nocturnal Misbehaviour)

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ensmallen_tests is a Catch v2.4.1 host application.
Run with -? for options

-------------------------------------------------------------------------------
SimpleAdaDeltaTestFunctionFMat
-------------------------------------------------------------------------------
/usr/ports/math/ensmallen/work/ensmallen-2.14.1/tests/ada_delta_test.cpp:66
...............................................................................

/usr/ports/math/ensmallen/work/ensmallen-2.14.1/tests/ada_delta_test.cpp:75: FAILED:
  REQUIRE( coordinates(1) == Approx(0.0f).margin(0.01) )
with expansion:
  -0.13114f == Approx( 0.0 )
```

OS: FreeBSD 12.1
clang-9

**Comment by UtR491** [comment]
I think you should take a look at #182.

**Comment by zoq** [comment]
@yurivict any chance you can check if https://github.com/mlpack/ensmallen/issues/182 works in some cases, if that's the case I'm going to bump the number of iterations.

**Comment by rcurtin** [comment]
@zoq I think bumping the iterations is the right thing to do here.  Let me know what you think; if you want, I can put together a PR, up to you. :+1:

**Comment by rcurtin** [comment]
Hey @yurivict, I believe this is fixed in #228.  So hopefully in the next ensmallen release you will not see this problem.  Thanks for the report! :+1:
